.github/workflows/ci.yml

```diff
@@ -35,22 +41,24 @@ jobs:
         run: yarn install
 
       - name: lint
-        run: yarn lint
+        run: yarn workspace ${{ matrix.workspace }} lint
 
   tests:
     name: Tests
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
+        workspace:
+          - ember-simple-auth
+          - test-app
```

Comment:
do we want to run these workspace tests for all Ember versions? I'm a bit worried the CI time will explode by doing that.

Reply:
I think we need to do this, otherwise we lose a bit of coverage as the tests in `packages/ember-simple-auth` are unit tests only and the ones in `packages/test-app` are application tests.